web/crux-ui/playwright.config.ts

```diff
@@ -17,8 +17,9 @@ const config: PlaywrightTestConfig = {
     command: 'npm run prod',
     url: baseURL,
     timeout: 60 * 1000, // 1 min
-    reuseExistingServer: !process.env.CI,
+    reuseExistingServer: true,
   },
+  workers: process.env.CI ? 4 : undefined,
```

Comment:
Why we need this?

Reply:
So we do not have to write code explicitly for CI, and we can use the same commands as we do during development.

Reply:
Okey but it would be better if we change `undefined` for some default numbers too.

Reply:
The `undefined` IS the default. It was shamelessly copied from the official docs: https://playwright.dev/docs/test-parallel#limit-workers

Reply:
I think it's fine this way